### ISSUE TITLE
Training ACGAN

### ISSUE THREAD

**Issue by ManiShiyam** [eriklindernoren/PyTorch-GAN#51]
Sorry this is not an issue. I have some questions regarding the implementation of ACGAN and training ACGAN. 
1. In your implementation the encoded label vector is multiplied with the noise vector and given as the input to the G. But shouldn't it be concatenated?
2. I am unclear about when to stop training. It seems (https://pytorch.org/tutorials/beginner/dcgan_faces_tutorial.html) the accuracy of the D for real vs fake images will be initially high for the real images, but low for the fake images. However, when I run ACGAN both are high initially (over 95%) and then reduces to ~70%. This accuracy does not go below 60%. So do you think my training is correct? Also I am unclear about when to stop training. May be this accuracy close to 50%, or the loss of the D converges, or loss of the G converges..?
